### PR TITLE
Update Gradle & MixinExtras to prepare for the upcoming 1.20.2 update

### PR DESCRIPTION
**Pull request by Fiz-Victor**
## Type of change

- [ ] Bug fix
- [ ] New feature

## Description

title

## Related issues

Mention any issues that this pr relates to.

# How Has This Been Tested?

Videos or screenshots of the changes if applicable.

# Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have added comments to my code in more complex areas.
- [X] I have tested the code in both development and production environments.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,8 +69,7 @@ dependencies {
     library "org.reflections:reflections:${project.reflections_version}"
     library("io.netty:netty-handler-proxy:${project.netty_version}") { transitive = false }
     library("io.netty:netty-codec-socks:${project.netty_version}") { transitive = false }
-    library "com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"
-    annotationProcessor "com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}"
+    annotationProcessor(include(implementation("com.github.LlamaLad7.MixinExtras:mixinextras-fabric:${project.mixin_extras_version}")))
```
Comment (arlomcwalter) [inline build.gradle:72]: why?

**Comment by Big-Iron-Cheems** [comment]
Quite sure this is already in my branch for 1.20.2 (might be missing the Gradle bump)

File: build.gradle (hunk 1 above)
Comment (Fiz-Victor) [reply build.gradle:72]: there might be some class conflict as https://github.com/RaphiMC/ImmediatelyFast/commit/d2394173c393ed7c5a1f5ef9a3945cc48f37b7a9 mentioned

Comment (Fiz-Victor) [reply build.gradle:72]: also see the document here https://gist.github.com/LlamaLad7/ec597b6d02d39b8a2e35559f9fcce42f
it has official usage

**Comment by Wide-Cat** [comment]
this is all gonna be included in the 1.20.2 update

**Pull request closed without merging**